draft-ietf-scim-device-model.mkd

```diff
@@ -994,10 +1007,46 @@ The following is the JSON representation of the Schema. Implementors MUST NOT va
 Security Considerations
 =======================
 
-Because provisioning operations are senstive, each client must be
+Because provisioning operations are sensitive, each client must be
 appropriately authenticated. Certain objects may be read-only or
 not visible based on who is connected.
 
+Each operation has its own security considerations.
+
+## Object Creation
+
+Object creation in this framedwork grants a device access to the
+infrastructure and will to a greater or lesser extent grant the
+infrastructure access to the device.  When IP-layer access is
+provisioned, then the access will be at the IP layer.  For non-IP
+layer access, such as provisioning of BLE devices, the access may
+be to the entire device.  The explicit grant is made when the
+credentials of the device are shared with the SCIM server.
+
+## Object Deletion
+
+Once granted, even if the object is removed, the server may or may not
+act on that removal.  The deletion of the object is a signal of intent
+by the application that it no longer expects the device to be on the
+network.  It is strictly up to the SCIM server and its back end policy
+to decide whether or not to revoke access to the infrastructure.  Any
+access grant by the device must be separately handled.
+
+## Read operations
+
+Read operations are necessary in order for an application to sync its
+state to know what devices it is expected to manage.  Only clients
+that have created objects or those they authorize SHOULD have the
+ability to read those objects.
+
+## Update Operations
+
+Update operations may be necessary if a device has been modified
+in some way.  The same access control policy for read operations is
+RECOMMENDED here.
+
```

Comment:
It may also be worth a Security Consideration section about logging. For the same reasons as read operation restrictions, logging of SCIM objects should be done carefully; SCIM servers, applications, and traffic inspection tools SHOULD avoid logging SCIM objects, or if logging must be done then treat the logs as sensitive and restrict read access to them.

Reply:
Done.